### DISCUSSION TITLE
private/how_to_pay_cursor

### DISCUSSION THREAD

**Discussion by giscus** [imarvinle/CSGuide#73] [Announcements]
# private/how_to_pay_cursor

支付宝升级Cursor 教程

https://csguide.cn/private/how_to_pay_cursor.html

**Comment by giga-drill** [comment]
前面的都和文章里写的一样，但是卡在最后一步。。

An error occurred while processing your payment. Try again later or with a different payment method.

**Reply by imarvinle** [reply]
可以换个干净的代理，换个浏览器试下

**Reply by giga-drill** [reply]
关掉梯子后可以了，感谢！

**Comment by xiaowangbb** [comment]
野卡跑路，是否有更安全的卡商

**Reply by imarvinle** [reply]
有需要用卡场景的 可以 getgpt.pro 找客服拿卡

**Reply by imarvinle** [reply]
[https://getgpt.pro/](https://getgpt.pro/i/how_to_pay_cursor)
